skill-tree/src/test/java/com/RDS/skilltree/integration/EndorsementsIntegrationTests.java

```diff
@@ -0,0 +1,520 @@
+                .body("data[0].endorserList[0].endorserId", equalTo("user-2"))
+                .body("data[0].endorserList[0].description", isA(String.class))
+                .body(
+                        "data[0].endorserList[0].userType",
+                        anyOf(equalTo("NORMAL_USER"), equalTo("SUPER_USER"), equalTo("MAVEN")));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Fetch all the endorsements given endorsement status")
+    public void testAPIReturnsEndorsementsGivenStatus() {
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getUserCookie())
+                        .queryParam("status", "PENDING")
+                        .get("/endorsements");
+
+        response
+                .then()
+                .statusCode(200)
+                .contentType("application/json")
+                .body("data", hasSize(1))
+                .body("data[0].endorseeId", equalTo("user-1"))
+                .body("data[0].skillName", equalTo("Java"))
+                .body("data[0].status", equalTo("PENDING"))
+                .body("data[0].endorsementType", anyOf(equalTo("POSITIVE"), equalTo("NEGATIVE")))
+                .body("data[0].endorsersList", hasSize(1))
+                .body("data[0].endorserList[0].endorserId", equalTo("user-2"))
+                .body("data[0].endorserList[0].description", isA(String.class))
+                .body(
+                        "data[0].endorserList[0].userType",
+                        anyOf(equalTo("NORMAL_USER"), equalTo("SUPER_USER"), equalTo("MAVEN")));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 400 on invalid endorsement status passed")
+    public void testAPIReturns400_OnInvalidStatusPassed() {
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getUserCookie())
+                        .queryParam("status", "APPROVAL")
+                        .get("/endorsements");
+
+        response
+                .then()
+                .statusCode(400)
+                .body("code", equalTo(400))
+                .body("message", equalTo("Invalid status passed"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 400 on invalid endorsementId passed")
+    public void testAPIReturns400_OnInvalidParameterPassed() {
+        String endorsementId = "randomId";
+        Response response =
+                given()
+                        .cookies(RestAPIHelper.getUserCookie())
+                        .pathParam("endorsementId", endorsementId)
+                        .patch("/endorsements");
+
+        response
+                .then()
+                .statusCode(400)
+                .body("code", equalTo(400))
+                .body("message", equalTo("Invalid endorsementId passed"));
+    }
+
+    @Test
+    @DisplayName("Return 200 on endorsements creation")
+    public void testAPIReturns201_OnEndorsementCreation() {
```

Comment:
Shouldn't `testAPIReturns201_OnEndorsementCreation` be `testAPIReturns200_OnEndorsementCreation` according to the display name `Return 200 on endorsements creation`?
Or vice versa, i.e, `Return 201 on endorsements creation`?

Reply:
Display name was wrong "Return 201 on endorsements creation", have updated the PR